.github/workflows/e2e-nvidia-l40s-x4.yml

```
@@ -295,10 +314,34 @@ jobs:
             --head-sha "${{ github.event.pull_request.head.sha }}" \
             --origin-repository "${{ github.repository }}"
```

You'll want it to be outputted here:

```suggestion
            --origin-repository "${{ github.repository }}"
            
          cat "${output_file}" >> "${GITHUB_STEP_SUMMARY}"
```